src/Polar/PolarStore.cpp

```diff
@@ -58,256 +58,206 @@ PolarStore::Item::ToPolarInfo() const
 }
 
 /**
- *  Note: new items should be added to bottom, otherwise saved index is lost 
- *  updated 2013/02/04 OH DAEC list 2012
```

Comment:
You must not reorder the items!
This code is really badly designed, but those polars are referenced by their index in this array.

Reply:
Removed index access to polar store entirely.